notebooks/02-ingestion/02-batch-ingestion.ipynb

```diff
@@ -631,6 +631,95 @@
     "\"\"\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "3e330808-696b-4278-be8e-269924cc8089",
+   "metadata": {},
+   "source": [
+    "## NEW in Druid 29.0 - System Fields for Batch Ingestion\n",
```

Comment:
As per elsewhere, avoid talking about specific versions in the body, except for the intro where the version requirement is given. Then the old notebook gets a version number suffix, as with lookups in 28,

Reply:
Applied new versioning strategy in the latest commit.